Serilog.Sinks.InfluxDB/Sinks/InfluxDB/InfluxDBSink.cs

```diff
@@ -107,6 +119,15 @@ public Task EmitBatchAsync(IEnumerable<Events.LogEvent> batch)
                     .Field(Fields.Version, Fields.Values.Version)
                     .Timestamp(logEvent.Timestamp.UtcDateTime, WritePrecision.Ms);
 
+                if(_includeContextAttributes)
```

Comment:
is the intent rather similar to open PR https://github.com/pada57/serilog-sinks-influxdb/pull/16

i would proprose we have two collections ExtendedTags and Fields.

in your case ContextAttributes are always added as Tags, cannot be a Field.
also "ContextAttributes" naming is rather confusing

would the other PR with some improvements of course would be workable solution in your use case ?